src/plugins/query_enhancements/common/constants.ts

```diff
@@ -48,6 +48,7 @@ export const URI = {
   PROMQL: '/_plugins/_promql',
   SQL: '/_plugins/_sql',
   ASYNC_QUERY: '/_plugins/_async_query',
+  DIRECT_QUERY_SYNC: '/_plugins/_directquery/_sync',
```

Comment:
```suggestion
  DIRECT_QUERY: {
    QUERY:  '/_plugins/_directquery/_query',
    RESOURCES:  '/_plugins/_directquery/_resources',
  },
```

if you pull the latest change, _sync should be renamed to _query.

currently resources API is actually
```
/_plugins/_directquery/{dataSources}/resources
```

should we make it `/_plugins/_directquery/_resources/{dataSource}` for consistency?

Reply:
> currently resources API is actually
>/_plugins/_directquery/{dataSources}/resources
>should we make it /_plugins/_directquery/_resources/{dataSource} for consistency?

Yeah `/_plugins/_directquery/_resources/{dataSource}` would be great here so it can match the endpoint structure.

Reply:
updated to make it `/_plugins/_directquery/_resources/{dataSource}`, you can pull and try it later. sample usage
```
GET /_plugins/_directquery/_resources/my_prometheus/api/v1/series?match[]={name=~'.+'}
```

Reply:
Thanks, just pulled the latest sql plugin changes and updated.